torch/onnx/utils.py

```diff
@@ -773,11 +772,11 @@ def _run_symbolic_function(g, n, inputs, env, operator_export_type=OperatorExpor
                     return g.op("Constant", value_t=n["value"])
                 if n.kindOf("value") == "s":
                     return g.op("Constant", value_s=n["value"])
-                elif n.kindOf("value") == "is":
-                    value = torch.stack([torch.tensor(v) for v in n["value"]]) if n["value"] else []
+                elif n.output().type().isSubtypeOf(ListType.ofInts()):
+                    value = torch.stack([torch.tensor(v) for v in n.output().toIValue()])
                     return g.op("Constant", value_t=value)
-                elif n.kindOf("value") == "fs":
-                    value = torch.stack([torch.tensor(v) for v in n["value"]]) if n["value"] else []
+                elif n.output().type().isSubtypeOf(ListType.ofFloats()):
+                    value = torch.stack([torch.tensor(v) for v in n.output().toIValue()])
```

Comment:
maybe merge this branch with `Ints` branch? the above change applies here as well to handle empty list case.

Reply:
I don't think this code path is executed for the caffe2 mnist failure, bc I uploaded it with a typo and the typo wasn't hit but the seg fault still happened